_posts/2021-03-08-scala-3-developers-preview-survey-results.md

```diff
@@ -216,8 +216,7 @@ compiler options. However we received 7 requests for the `-Xlint` options,
 
 ### Communication
 ![Communication result survey](/resources/img/blog/scala-3-developers-preview-survey-results/communication-survey.svg)
-We asked participants the question “How frequently do you use the following c
```

Comment:
This word was unfortunately split across a line...

Personally, I'd encourage splitting paragraphs at least by sentence as a sentence is a unit in English and it makes sense to reason about changes to sentence, but that's beyond the scope of this PR.